xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/jackson/modules/LedgerIndexSerializer.java

```diff
@@ -2,38 +2,26 @@
 
 import com.fasterxml.jackson.core.JsonGenerator;
 import com.fasterxml.jackson.databind.SerializerProvider;
-import com.fasterxml.jackson.databind.ser.std.StdScalarSerializer;
-import org.xrpl.xrpl4j.model.client.common.LedgerIndex;
-import org.xrpl.xrpl4j.model.transactions.Address;
+import com.fasterxml.jackson.databind.ser.std.StdSerializer;
+import org.xrpl.xrpl4j.model.client.specifiers.LedgerIndex;
 
 import java.io.IOException;
 
 /**
  * Custom Jackson serializer for {@link LedgerIndex}s.
  */
-public class LedgerIndexSerializer extends StdScalarSerializer<LedgerIndex> {
+public class LedgerIndexSerializer extends StdSerializer<LedgerIndex> {
```

Comment:
I would deprecate this too since it will go away with `LedgerIndex` eventually.

Reply:
This is also for the new `LedgerIndex` class.

I deprecated the old `LedgerIndex` as well as its jackson modules.